### PR TITLE
Conformance results for v1.25/kind

### PR DESCRIPTION
**Pull request by aojea**
### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [ ] If this is a new entry, have you submitted a signed [participation form](https://github.com/cncf/k8s-conformance/tree/master/participation-form)?
* [x] Did you include the product/project logo in SVG, EPS or AI format?  
* [x] Does your logo clearly state the name of the product/project and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [x] If your product/project is open source, did you include the `repo_url`?
* [x] Did you copy and paste the [installation and configuration instructions](https://github.com/cncf/k8s-conformance/blob/master/faq.md#can-i-provide-a-link-to-the-installation-directions) into the README.md file in addition to linking to them?

/cc @BenTheElder

### CODE REVIEW

File: v1.25/kind/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+`
+2. Create a Kubernetes cluster
+
+The Kubernetes version depends on the node image chosen, the list with the supported
+Kubernetes versions and its associated image is published with the release notes:
+
+https://github.com/kubernetes-sigs/kind/releases
+
+```
+./kind create cluster --image kindest/node:v1.25.0
```
Comment (BenTheElder) [inline v1.25/kind/README.md:21]: Let's pin the image for reproducibility?
This flag can also be skipped on v0.15.0 as the current 1.25.0 image is the default image for the tool.

Comment (aojea) [reply v1.25/kind/README.md:21]: let's keep it this way, I think that we just have to demonstrate we are not cheating, I don't think that this instructions are meant to be used by users

**Comment by aojea** [comment]
it may look for certain suonoboy output, I'll redo it with it

File: v1.25/kind/README.md (hunk 1 above)
Comment (BenTheElder) [reply v1.25/kind/README.md:21]: It's meant to be possible to reproduce, and it's not best practice to use kind node images without pinning them per our own directions.

dropping the image flag would be even simpler.

**Comment by taylorwaggoner** [comment]
You are now Certified Kubernetes

**Pull request merged**